src/components/FindMovie/FindMovie.js

```diff
@@ -1,55 +1,110 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './FindMovie.scss';
-
+import PropTypes from 'prop-types';
 import { MovieCard } from '../MovieCard';
-import movies from '../../api/movies.json';
-
-export const FindMovie = () => (
-  <>
-    <form className="find-movie">
-      <div className="field">
-        <label className="label" htmlFor="movie-title">
-          Movie title
-        </label>
-
-        <div className="control">
-          <input
-            type="text"
-            id="movie-title"
-            placeholder="Enter a title to search"
-            className="input is-danger"
-          />
+
+import { getMovie } from '../../api';
+
+export const FindMovie = ({ addMovie }) => {
+  const [value, setValue] = useState('');
+  const [movie, setMovie] = useState('');
+  const [searchError, setSearchError] = useState(false);
+
+  const onChangeValue = (event) => {
+    setValue(event.target.value);
+    setSearchError(false);
+    setMovie('');
+  };
+
+  const searchMovie = () => {
+    if (value) {
+      getMovie(value)
+        .then((result) => {
+          if (result.Response === 'False') {
+            setSearchError(true);
+            setMovie('');
+          } else {
+            setMovie({
+              title: result.Title,
+              description: result.Plot,
+              imgUrl: result.Poster,
+              imdbId: result.imdbID,
+              imdbUrl: `https://www.imdb.com/title/${result.imdbID}`,
+            });
+          }
+        });
+    }
+  };
+
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    addMovie(movie);
+    setValue('');
+    setMovie('');
+  };
+
+  return (
+    <>
+      <form
+        className="find-movie"
+        onSubmit={handleSubmit}
+      >
+        <div className="field">
+          <label className="label" htmlFor="movie-title">
+            Movie title
+          </label>
+
+          <div className="control">
+            <input
+              type="text"
+              id="movie-title"
+              placeholder="Enter a title to search"
+              className={`input ${searchError ? 'is-danger' : ''}`}
```

Comment:
```suggestion
              className={`input ${searchError && 'is-danger'}`}
```
or even better to use classnames library here

Reply:
Fixed